qabelbox/src/androidTest/java/de/qabel/qabelbox/ui/LogoutUITest.java

```diff
@@ -0,0 +1,100 @@
+package de.qabel.qabelbox.ui;
+
+import android.content.Context;
+import android.content.Intent;
+import android.support.test.InstrumentationRegistry;
+import android.support.test.espresso.intent.Intents;
+import android.support.test.espresso.intent.rule.IntentsTestRule;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import de.qabel.qabelbox.QabelBoxApplication;
+import de.qabel.qabelbox.R;
+import de.qabel.qabelbox.TestConstants;
+import de.qabel.qabelbox.activities.MainActivity;
+import de.qabel.qabelbox.communication.URLs;
+import de.qabel.qabelbox.config.AppPreference;
+import de.qabel.qabelbox.services.LocalQabelService;
+import de.qabel.qabelbox.ui.helper.UIBoxHelper;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.contrib.DrawerActions.openDrawer;
+import static android.support.test.espresso.intent.matcher.IntentMatchers.hasComponent;
+import static android.support.test.espresso.intent.matcher.IntentMatchers.hasFlag;
+import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+import static org.hamcrest.Matchers.allOf;
+import static org.hamcrest.Matchers.empty;
+import static org.hamcrest.Matchers.not;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.hamcrest.Matchers.nullValue;
+import static org.junit.Assert.assertThat;
+
+public class LogoutUITest {
+    public static final String ACCOUNT_NAME = "account_name";
+    public static final String ACCOUNT_E_MAIL = "accountmail@example.com";
+    IntentsTestRule<MainActivity> mainActivityActivityTestRule =
+            new MainActivityWithoutFilesFragmentTestRule();
+    private AppPreference appPreference;
+    private LocalQabelService service;
+
+
+    @Before
+    public void setUp() {
```

Comment:
The problem with that are the different test rules, but I could extract a super class.